models/stics/inst/prof.mod

```diff
@@ -0,0 +1,4 @@
+2
```

Comment:
I'm guessing this filename is set by Stics and not under our control, but if it can be changed: I'm used to assuming `*.mod` are compiled Fortran modules that should be deleted any time I want a clean compilation. Is there a risk of someone naively trying something like `find pecan/ -name '*.mod' -delete` and then wondering why Stics broke?

Reply:
yes the filename is what STICS expects and I'm not sure if it can be changed. Stics will break for many other reasons before it comes to this 😄 but joking aside, Stics writes out error logs and I'm sure it will say that it cannot find `X.mod` file in such a case, which should help find the cause